pliers/extractors/text.py

```diff
@@ -313,3 +335,38 @@ def _extract(self, stim):
         features = ['sentiment_' + k for k in scores.keys()]
         return ExtractorResult([list(scores.values())], stim, self,
                                features=features)
+
+class SpaCyExtractor(TextExtractor):
+    
+    def __init__(self, model='en_core_web_sm', features=None):
+        #verify_dependencies(['spacy'])
```

Comment:
This is the right approach; you can uncomment this line and `spacy` will be available in the namespace (assuming you've imported with `attempt_to_import` as mentioned above).